alpha/alpha-algo/src/main/java/org/neo4j/graphalgo/results/InfluenceMaximizationResult.java

```diff
@@ -0,0 +1,48 @@
+package org.neo4j.graphalgo.results;
+
+import org.neo4j.graphalgo.result.AbstractResultBuilder;
+
+public class InfluenceMaximizationResult
+{
+    public final long nodeId;
+    public final double spreadGain;
+
+    public InfluenceMaximizationResult( long nodeId, double spreadGain )
+    {
+        this.nodeId = nodeId;
+        this.spreadGain = spreadGain;
+    }
```

Comment:
I change that to spread instead of spreadGain.
Let's say k = 3. The first node is 'Alice' with spread = 1.5, the second node is 'Ethan' with score 2.7 (iff Alice is active), the third node is 'Michael' with score 3.8 (iff Alice and Ethan are active).
Should I change the spread according Alice: 1.5, Ethan: 1.2, Michael 1.1 ?